src/main/java/com/teragrep/rlp_01/client/SSLContextSupplier.java

```diff
@@ -0,0 +1,13 @@
+package com.teragrep.rlp_01.client;
+
+import com.teragrep.rlp_01.pool.Stubable;
+
+import javax.net.ssl.SSLContext;
+import java.util.function.Supplier;
+
+/**
+ * Wrapper for SSLContext because such do not have stubness property
+ */
+public interface SSLContextSupplier extends Supplier<SSLContext>, Stubable {
```

Comment:
Should this filename indicate it is an interface like IRelpConnection does

Reply:
IRelpConnection is just to avoid the mixing with RelpConnection, which should be actually RelpConnectionImpl. this would require major version up as it will be incompatible so not doing it here.